packages/web-react/docs/DocsSections.tsx

```diff
@@ -17,11 +19,13 @@ const DocsSection = ({ children, title, tag }: DocsSectionProps) => (
         </Tag>
       )}
     </h2>
-    {children}
+    {hasStack ? <div className={`docs-Stack docs-Stack--${stackAlignment}`}>{children}</div> : children}
```

Comment:
Do we have the `DocsStack` component?

Reply:
Created new `DocsStack` component in react and twig